run/archiveloop

```diff
@@ -258,6 +258,24 @@ function mount_and_fix_errors_in_music_file () {
   fi
 }
 
+function trim_free_space() {
+  local mount_point="$1"
+  local mount_exists=true
+
+  # Make sure the partition is mounted.
+  findmnt --mountpoint "$mount_point" > /dev/null || mount_exists=false
+  if [ "$mount_exists" = true ]
```

Comment:
I think these 5 lines can be simplified to just:
```
if findmnt --mountpoint "$mount_point" > /dev/null
```

Reply:
Fixed.